scripts/install.sh

```diff
@@ -2,6 +2,6 @@
 set -e
 set -x
 
-yarn add react-native-linear-gradient
+yarn add react-native-linear-gradient@2.7.3
```

Comment:
React native dependencies need to be added to package.json. packager will link then automatically. No need to use the install script for that